roles/edpm_telemetry/tasks/configure.yml

```diff
@@ -30,6 +30,7 @@
 - name: Render ceilometer config files
   tags:
     - edpm_telemetry
+  become: true
```

Comment:
become should not need to be used here as the directory should be owned by the current user.